src/app/common/PolicyTypeTag.vue

```diff
@@ -1,9 +1,9 @@
 <template>
   <span class="policy-type-tag">
     <img
-      v-if="policy.iconUrl !== null"
+      v-if="POLICY_ICON[props.policyType]"
       class="policy-type-tag-icon"
-      :src="policy.iconUrl"
+      :src="POLICY_ICON[props.policyType]"
```

Comment:
What if there isn't one?

Reply:
If there isn’t one, we don’t show an icon at all.